### PR TITLE
Add Ubuntu eoan package

### PR DESCRIPTION
**Pull request by fengzhou-msft**
All current Ubuntu releases can be checked [here](https://wiki.ubuntu.com/Releases).

Will modify Release Pipeline later to reflect this change.

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -825,6 +825,37 @@ jobs:
...
+
+
+    - task: Bash@3
+      displayName: 'Build Ubuntu Eoan Package'
+      inputs:
+        targetType: 'filePath'
+        filePath: scripts/release/debian/pipeline.sh
+      env:
+        DISTRO: eoan
+        DISTRO_BASE_IMAGE: ubuntu:eoan
```
Comment (haroldrandom) [inline azure-pipelines.yml]: suggest to refactor as:

```YAML
- bash: scripts/release/debian/pipeline.sh
  displayName: 'Build Ubuntu Eoan Package'
  env:
    DISTRO: eoan
    DISTRO_BASE_IMAGE: ubuntu:eoan
```

**Comment by haroldrandom** [comment]
A suggestion, jobs related to image build across Linux distributions in azure-pipeline.yml can refactor as this doc [Create a multi-platform pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops).

It do the same job as current but simplify.

File: azure-pipelines.yml (hunk 1 above)
Comment (fengzhou-msft) [reply azure-pipelines.yml]: updated

**Comment by fengzhou-msft** [comment]
> A suggestion, jobs related to image build across Linux distributions in azure-pipeline.yml can refactor as this doc [Create a multi-platform pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops).
> 
> It do the same job as current but simplify.

Currently the task is running on the same ubuntu system but with different variable(DISTRO, DISTRO_BASE_IMAGE) for docker image, can consider your method without using docker in future refactor.

**Comment by haroldrandom** [comment]
> > A suggestion, jobs related to image build across Linux distributions in azure-pipeline.yml can refactor as this doc [Create a multi-platform pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops).
> > It do the same job as current but simplify.
> 
> Currently the task is running on the same ubuntu system but with different variable(DISTRO, DISTRO_BASE_IMAGE) for docker image, can consider your method without using docker in future refactor.

No related to docker. Like this:
```YAML
strategy:
  matrix:
    ubuntu1604:
      imageName: 'ubuntu-16.04'
    ubuntu1804:
      imageName: ''ubuntu-18.04'
    ubuntu1904:
      imageName: ''ubuntu-19.04'
    ubuntu1910:
      imageName: ''ubuntu-19.10'
```

No hurry.

**Comment by fengzhou-msft** [comment]
> > > A suggestion, jobs related to image build across Linux distributions in azure-pipeline.yml can refactor as this doc [Create a multi-platform pipeline](https://docs.microsoft.com/en-us/azure/devops/pipelines/get-started-multiplatform?view=azure-devops).
> > > It do the same job as current but simplify.
> > 
> > 
> > Currently the task is running on the same ubuntu system but with different variable(DISTRO, DISTRO_BASE_IMAGE) for docker image, can consider your method without using docker in future refactor.
> 
> No related to docker. Like this:
> 
> ```yaml
> strategy:
>   matrix:
>     ubuntu1604:
>       imageName: 'ubuntu-16.04'
>     ubuntu1804:
>       imageName: 'ubuntu-18.04'
>     ubuntu1904:
>       imageName: 'ubuntu-19.04'
>     ubuntu1910:
>       imageName: 'ubuntu-19.10'
> pool:
>   vmImage: $(imageName)
> ```
> 
> No hurry.

Current tasks for building various ubuntu packages are all running on a vm of `ubuntu-16.04`, but with different docker images. To switch to your way which looks more neat, needs some refactoring.

**Pull request merged**